website/docs/how-to/how-to-create-and-manage-user-groups.md

```diff
@@ -0,0 +1,73 @@
+---
+title: How to create and manage user groups
+---
+
+:::info availability User groups are available to Unleash Enterprise users since **Unleash 4.14**. :::
+
+This guide takes you through how use user groups to manage permissions on your projects. User groups allow you to manage large groups of users more easily than assigning roles directly to those users.
```

Comment:
Nice! Can you add a link to the section on user groups in the reference docs here. Something like "refer to the section on user groups in the RBAC documentation for more information"?

Reply:
Good call! Done!